.appveyor/set_paths.bat

```diff
@@ -0,0 +1,17 @@
+@echo on
+set old_dir="%CD%"
+cd %~dp0..\..
+set CURL_BIN=curl
+set UNZIP_BIN=unzip
+set MSYSDIR=C:\MinGW\msys\1.0
+set PERL_BIN=perl
+set CMAKE_BIN=cmake
+set BUILDDIR=%CD%
+set BINARIESDIR=%BUILDDIR%\binaries
```

Comment:
In one of the above comment you mention about SVN dependency. I don't see any reference here.